src/Data/Schematic/Generator.hs

```diff
@@ -0,0 +1,118 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Is there a reason to have these instances separately? It's possible to extract the definitions from `Data.Schematic.Schema` to something like `Data.Schematic.Schema.Types` or use mutual recursive modules so we won't have these orphan instances https://downloads.haskell.org/~ghc/7.0.3/docs/html/users_guide/separate-compilation.html